src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToStartHandler.java

```diff
@@ -13,32 +16,23 @@
 import uk.gov.hmcts.reform.sscs.ccd.domain.*;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
 import uk.gov.hmcts.reform.sscs.service.RegionalProcessingCenterService;
-import uk.gov.hmcts.reform.sscs.service.servicebus.HearingMessagingServiceFactory;
 
 @Service
+@RequiredArgsConstructor
 @Slf4j
-public class ReadyToListAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
```

Comment:
I'm not sure about this changing to an about to start handler. It would be safer to keep it as an about to submit handling the GAPs case. If not GAPs then the submitted handler handles it - to minimise change.